packages/analytics-js/src/state/slices/nativeDestinations.ts

```diff
@@ -1,36 +1,26 @@
 import { Signal, signal } from '@preact/signals-core';
-import {
-  ClientIntegration,
-  Destination,
-  InitialisedIntegration,
-  IntegrationInstance,
-  IntegrationOpts,
-} from '@rudderstack/analytics-js/state/types';
+import { Destination, IntegrationOpts } from '@rudderstack/analytics-js/state/types';
 
 export type NativeDestinationsState = {
-  destinations: Signal<Destination[]>;
-  activeIntegrations: Signal<ClientIntegration[]>;
+  configuredDestinations: Signal<Destination[]>;
+  activeDestinations: Signal<Destination[]>;
   loadOnlyIntegrations: Signal<IntegrationOpts>;
-  clientIntegrationObjects: Signal<IntegrationInstance[] | undefined>;
-  loadedIntegrationScripts: Signal<string[]>;
-  failedIntegrationScripts: Signal<string[]>;
+  failedDestinations: Signal<Destination[]>;
   loadIntegration: Signal<boolean>;
   integrationsData: Signal<Record<string, any>>;
-  initialisedIntegrations: Signal<Record<string, InitialisedIntegration>>;
-  clientIntegrationsReady: Signal<boolean>;
+  initializedDestinations: Signal<Destination[]>;
+  clientDestinationsReady: Signal<boolean>;
 };
 
 const nativeDestinationsState: NativeDestinationsState = {
-  destinations: signal([]),
-  activeIntegrations: signal([]),
+  configuredDestinations: signal([]),
+  activeDestinations: signal([]),
   loadOnlyIntegrations: signal({}),
-  clientIntegrationObjects: signal(undefined),
-  loadedIntegrationScripts: signal([]),
-  failedIntegrationScripts: signal([]),
+  failedDestinations: signal([]),
   loadIntegration: signal(true),
```

Comment:
What is the usage of this signal `loadIntegration`?

Reply:
This is used by the integrations code. It'll be used to decide whether to load the native SDK. It is helpful for scenarios where the customer is already instrumenting the native SDK on their app and wants RS SDK to just forward the events to it.